packages/core/README.md

```diff
@@ -232,6 +232,38 @@ import {LogLevel} from '@adobe/react-native-aepcore';
 MobileCore.setLogLevel(LogLevel.VERBOSE);
 ```
 
+- #### trackAction
+
+Track event actions that occur in your application.
+
+**Syntax**
+
+```typescript
+trackAction(action?: string, contextData?: Record<string, string>)
+```
+
+**Example**
+
+```typescript
+MobileCore.trackAction('loginClicked', {"customKey", "value"});
```

Comment:
nit: for consistency can we use double quotes for all strings - same for both trackState and trackAction samples